### PR TITLE
Adds Mad Max to the Livesplit.AutoSplitters.xml

### PR DESCRIPTION
**Pull request by plyd823**
If you are adding or modifying an Auto Splitter, please fill out this checklist:
- [X] My Auto Splitter does not contain any malicious functionality and I'm entirely responsible for any damages myself. (Any kind of abuse in an Auto Splitter will result in an immediate ban.)
- [X] I am not replacing an existing Auto Splitter by a different author, or I have received permission from the author to replace the existing Auto Splitter.
- [X] The Auto Splitter has an open source license that allows anyone to fork and host it.

### CODE REVIEW

**Comment by just-ero** [comment]
@wooferzfg How did this pass checks?

**Comment by wooferzfg** [comment]
The checks explicitly rule out known invalid URL formats. We don't have a way to check every single  possible URL and make sure it's correct (but feel free to improve the tests)

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -10,6 +10,16 @@
...
+    <AutoSplitter>
+        <Games>
+            <Game>Mad Max (2015)</Game>
+        </Games>
+        <URLs>
+            <URL>https://github.com/plyd823/MadMaxAutosplitter</URL>
+        </URLs>
+        <Type>Script</Type>
+        <Description>Autosplitter by Plyd</Description>
+    </AutoSplitter>
```
Comment (just-ero) [inline LiveSplit.AutoSplitters.xml:22]:
```suggestion
    <AutoSplitter>
        <Games>
            <Game>Mad Max</Game>
        </Games>
        <URLs>
            <URL>https://github.com/plyd823/MadMaxAutosplitter/raw/main/MadMaxAutosplitter.asl</URL>
        </URLs>
        <Type>Script</Type>
        <Description>Autosplitter by Plyd</Description>
    </AutoSplitter>
```

**Comment by plyd823** [comment]
Committed the link change, let me know if you need anything else from me! :)

**Pull request merged**